### PR TITLE
feat: added Ad page

### LINKED ISSUE #3: 🚀 Feature: Add an "ad" page
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/philly-js-club-site/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

As [requested in Slack](https://phillyjs.slack.com/archives/C04J3DT5NCF/p1681953713175109), let's make this image dynamically rendered:

Screenshot of page with PhillyJS logo on the left and 'Philly JS Club April 2023' on the right

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## PR Checklist

- [x] Addresses an existing open issue: fixes #3
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/philly-js-club-site/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/philly-js-club-site/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Edit: test on URLs like http://localhost:3000/ad?month=april&year=2023

Screenshot of the requested page with light-on-dark text

Screenshot of the requested page with dark-on-light text

Co-authored-by: Zae Holland <zae.holland@cleanchoiceenergy.com>

Co-authored-by: Chasen Stark <chasenstark@gmail.com>

### CODE REVIEW

**Comment by tjwds** [comment]
@chasenstark's going to take a look at this and merge it!

File: app/shared.css [hunk 1]
```diff
@@ -208,3 +208,90 @@ Let us know if you think you've found one we'd like. 😏
...
+}
+
+.ad-text {
+	display: flex;
+	flex-direction: column;
+	gap: 2rem;
+	text-align: center;
+}
+
+.ad-text h1 {
```
Comment (chasenstark) [inline app/shared.css:238]: What do you think about adding `line-height: .85` to this selector?

**Review by chasenstark — APPROVED** [review]
@JoshuaKGoldberg This is _great_! Thanks for doing this. 🙏

File: app/shared.css (hunk 1 above)
Comment (JoshuaKGoldberg) [reply app/shared.css:238]: Yeah I like it, thanks!

**Pull request merged**